src/AutoMapper/Configuration/PathConfigurationExpression.cs

```diff
@@ -39,14 +44,6 @@ public void MapFromUntyped(LambdaExpression sourceExpression)
 
         public void Configure(TypeMap typeMap)
         {
-            //var destMember = DestinationMember;
```

Comment:
This was a reminder that I left smth out, but I guess somebody will eventually complain.